src/Microsoft.AspNet.Facebook/Microsoft.AspNet.Facebook.csproj

```diff
@@ -27,7 +27,7 @@
       <HintPath>..\..\packages\Facebook.6.4.2\lib\net45\Facebook.dll</HintPath>
     </Reference>
     <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
Should 6.0.0.0 change to a new version number?

Reply:
Yes @philipwolfe. I should have marked this as a draft from the get-go.